core-api/src/main/java/org/silverpeas/core/calendar/repository/CalendarEventRepository.java

```diff
@@ -24,7 +24,9 @@
 
 package org.silverpeas.core.calendar.repository;
 
+import org.silverpeas.core.admin.user.model.User;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.silverpeas.core.admin.user.model.User'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)